.travis.yml

```diff
@@ -1,14 +1,21 @@
 language: python
-dist: trusty
-sudo: false
 cache:
   pip: true
 env:
   - C7N_TEST_WORKERS=8
 
-python:
-  - "2.7"
-  - "3.6"
+matrix:
+    include:
+        - python: 2.7
+          dist: trusty
+          sudo: false
+        - python: 3.6
+          dist: trusty
+          sudo: false
+        - python: 3.7
+          dist: xenial
+          sudo: true
```

Comment:
do we need the sudo?

Reply:
Yeah. I am a bit in shock on that too. I think 3.7 is still being worked into Travis. 
https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-401756442

Sounds like someone got it to work on xenial without sudo so I will try that now.
https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-403142521

Sounds like its an issue of a missing openssl library on trusty:
https://docs.travis-ci.com/user/languages/python/#Development-releases-support

Travis Issue:
https://github.com/travis-ci/travis-ci/issues/9069